tst/regression/regression_test.py

```diff
@@ -17,7 +17,7 @@
 from subprocess import call
 import shutil
 import glob
-sys.path.append("../../external/parthenon/scripts/python/packages/parthenon_tools")
+sys.path.insert(0,"../../external/parthenon/scripts/python/packages/parthenon_tools")
```

Comment:
The `append`ed `parthenon_tools` would be ignored if a (potentially different) version of `parthenon_tools` was installed locally.